### PR TITLE
Set node property even when equal to default prop value

### PR DESCRIPTION
**Pull request by waynebaylor**
Fix for #257 

Added a test to highlight the issue.

My fix might be heavy handed, so totally open to feedback.

### CODE REVIEW

**Comment by brainkim** [comment]
Okay. I am awake and a little hung over. Thank you for this PR and sorry if this is blocking you. I am trying to stay away from the laptop so I wanted to demonstrate my appreciation for your help by providing my thoughts impromptu and hoping you can figure out the rest.

My initial thoughts are: This is a situation involving the initial render. This means that we're likely working with new DOM nodes which have no attributes or properties yet. So maybe we could do a check based on that? That conditional which you are trying to change does some stuff which I can't recall right now. Hot code. Live wire stuff.

By not setting properties when they are triple equal, ah yes now I remember, the main thing we are trying to stop is unexpected form input mutations? How do I say this plainly without looking up actual issues? Or maybe that's the opposite for this situation?

I guess, in a general sense, DOM nodes can change independently of rendering, so we're trying to treat it with kid gloves. This specific branch was likely put in place because it is important not to change attributes for some reason. The GitHub mobile view is not letting me see the full layers  of indentation.

TL;DR can we disable the branch for just the _initial_ render???

**Comment by brainkim** [comment]
Also thank you for the test!

**Comment by brainkim** [comment]
Perfecto! Thank you so much!
